src/main/relation/projection_relation.cpp

```diff
@@ -14,7 +14,7 @@ ProjectionRelation::ProjectionRelation(shared_ptr<Relation> child_p,
 			throw ParserException("Aliases list length must match expression list length!");
 		}
 		for (idx_t i = 0; i < aliases.size(); i++) {
-			expressions[i]->alias = aliases[i];
+			expressions[i]->SetAlias(aliases[i]);
```

Comment:
I wonder if we should also make the alias immutable and add a constructor that takes another expression and a new alias instead, but again, next PR